test/fixtures/poro.rb

```diff
@@ -38,9 +38,16 @@ class ProfilePreviewSerializer < ActiveModel::Serializer
 
 Post     = Class.new(Model)
 Like     = Class.new(Model)
-Author   = Class.new(Model)
+Author   = Class.new(Model) do
+  # To confirm cache_key is set using updated_at and cache_key option passed to cache
+  undef_method :cache_key
+end
 Bio      = Class.new(Model)
 Blog     = Class.new(Model)
+Article  = Class.new(Model) do
+  # To confirm error is raised when cache_key is not set and cache_key option not passed to cache
+  undef_method :cache_key
+end
```

Comment:
@kevintyll yup.  I'd love to get rid of most of it, but some is useful, and maybe we can just create a dummy rails app for testing like in https://github.com/rails-api/active_model_serializers/pull/1412 

I think that would be better in the long run, e.g. how rails plugins create a `dummy` app